runtime/src/bank.rs

```diff
@@ -1616,6 +1622,16 @@ impl Bank {
             .map(|(acc, _slot)| acc)
     }
 
+    fn get_sysvar_account(&self, pubkey: &Pubkey) -> Option<Account> {
```

Comment:
write tests for this; this is indirectly needed to fix [this updated test](https://github.com/solana-labs/solana/pull/7892/files#diff-a7549f152920d85fb44e6a784b8e5e1fR3726).

Reply:
done; https://github.com/solana-labs/solana/pull/7892/commits/11c28cfcc1c4ccbaaf927d8dd12a290c8b1b1899